bot/admin/admin.py

```diff
@@ -27,10 +27,8 @@ def __init__(self, bot):
         self.bot = bot
         self._db_connector = DatabaseConnector(constants.DB_FILE_PATH, constants.DB_INIT_SCRIPT)
 
-        self.guild = bot.get_guild(int(constants.SERVER_ID))
-
         # Channel instances
-        self.ch_bot = self.guild.get_channel(int(constants.CHANNEL_ID_BOT))
+        self.ch_bot = bot.get_guild(int(constants.SERVER_ID or None)).get_channel(int(constants.CHANNEL_ID_BOT))
```

Comment:
Why use None here if you have already specified a value in constants.py

Reply:
Because of the kind of special way argument passing has been implemented in Python, especially when one is using built-in functions, the interpreter.... I simply forgot to remove it after testing some stuff... okay?